tests/literals/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -49,5 +59,16 @@ $content = sprintf(<<<'EOF'
 foo
 EOF
 );
+$content = array(
+    <<<EOF
+foo
+EOF
+    ,
+    "bar"
+);
+$content = array(<<<'EOF'
+foo
+EOF
+, "bar");
```

Comment:
Looks we have `)` on line with item. Ugly?

Reply:
Yeah, there is definitely still room for improvement. Let's do that in a separate PR though.